content/08-cardano-sidechains/03-example-evm-sidechain/06-transferring-tokens.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Transferring tokens
+metaTitle: Transferring tokens
+---
+
+This tutorial will show you how to transfer `SC_Tokens` between the main chain and a sidechain.
+
+## Prerequisites
+
+1. We will use the **sidechain-cli** and the **sc-evm-cli** components within this tutorial. Download the binaries and add them to your shell’s path: https://github.com/input-output-hk/sidechains-tooling.
+2. Download the **bech32** conversion tool binary and add to your shell's path: https://github.com/input-output-hk/bech32.
+3. If you have not done so already, create and fund your main chain and sidechain accounts: [Create and fund your accounts](/cardano-sidechains/example-evm-sidechain/create-and-fund-accounts).
+
+# Transfering tokens from a sidechain to the main chain
+
+Transferring `SC_Token` from a sidechain to the main chain will require interfacing with the bridge smart contract to lock your desired amount of `SC_Token` to be claimed on the main chain. In this guide, you will use RemixIDE to interface with the bridge contract.
+
+
+### Step 01 - Setup contract interface in RemixIDE
+
+- Go to https://remix.ethereum.org/
+
+- Create `/contracts/bridge.abi` with the following contents: [bridge.abi](https://github.com/input-output-hk/sidechains-tooling/blob/master/bridge.abi)
+
+*Note: Make sure your Remix-compatible Web3 provider is configured to the EVM sidechain testnet. Eg: MetaMask.*
+
+- In Remix, go to **Deploy and Run transactions** > select **Injected Provider (MetaMask)** for the environment > Load contract from address (`0x696f686b2e6d616d626100000000000000000000`)
+
+Your RemixIDE will look similar to this with a contract interface: ![Remix IDE](../../images/remix-ide.png)
+
+### Step 02 - Lock SC_Token
+
```

Comment:
IDK why, this step appears as 03 on https://deploy-preview-449--magenta-clafoutis-3ca66e.netlify.app/cardano-sidechains/example-evm-sidechain/transferring-tokens

Reply:
This is outdated now. Should be resolved now.